serverless.yml

```diff
@@ -0,0 +1,122 @@
+service: klau-drive
+frameworkVersion: '3'
+
+plugins:
+  - serverless-localstack
+
+provider:
+  name: aws
+  runtime: python3.9
+  stage: dev
+  region: us-east-1
+  memorySize: 128
+  timeout: 20
+  environment:
+    CONTENT_BUCKET_NAME:
+      Ref: contentBucket
+    CONTENT_METADATA_TB_NAME:
+      Ref: fileMetaTable
+    USER_TB_NAME:
+      Ref: userTable
+    ENDPOINT: ${env:ENDPOINT}
+  iamRoleStatements:
+    - Effect: Allow
+      Action:
+        - dynamodb:DescribeTable
+        - dynamodb:Query
+        - dynamodb:Scan
+        - dynamodb:GetItem
+        - dynamodb:PutItem
+        - dynamodb:UpdateItem
+        - dynamodb:DeleteItem
+      Resource:
+        - "Fn::GetAtt": [userTable, Arn]
+    - Effect: Allow
+      Action:
+        - dynamodb:DescribeTable
+        - dynamodb:Query
+        - dynamodb:Scan
+        - dynamodb:GetItem
+        - dynamodb:PutItem
+        - dynamodb:UpdateItem
+        - dynamodb:DeleteItem
+      Resource:
+        - "Fn::GetAtt": [fileMetaTable, Arn]
+
+package:
+  individually: True
+  patterns:
+    - '!**/**'
+    - 'src/lambdas/common.py'
+
+functions:
+  listFiles:
+    handler: src/lambdas/lambda_list_files.lambda_list_files
+    package:
+      patterns:
+        - 'src/lambdas/lambda_list_files.py'
+    events:
+      - http:
+          path: file
+          method: get
+          # TODO: enable cors later, better security

```

Comment:
I think we won't need this. CORS is just what the browser uses to enforce the same origin. Non-browser clients are unaffected by cors settings ([see here](https://stackoverflow.com/questions/43432743/will-cors-policy-prevent-resource-access-from-non-browser-requests)), and ours is a desktop gui client so we're don't have to worry about this.

Reply:
Maybe. I think browser clients can still use our api to do an attack because of missing cors. Will leave this for later, after research